web/v1/redshift/sql-runner/sql/custom/01-indicative-export/03-indicative-export-upsert.sql

```diff
@@ -0,0 +1,11 @@
+-- 3. DELETE - INSERT to production (and optionally drop the temp table)
```

Comment:
Not sure there is an option to drop the temp table?!

Reply:
Good point XD I copied from an example. Will clean up